timescaledb/how-to-guides/page-index/page-index.js

```diff
@@ -474,6 +471,27 @@ module.exports = [
           }
         ]
       },
+      {
+        href: "analytics",
+        children: [
+          {
+            title: "About Timescale Analytics",
```

Comment:
Does this title need to change now @avthars / @davidkohn88 / @JLockerman ??

Reply:
@ryanbooz we made out edits in https://github.com/timescale/docs/pull/149 b/c we weren't sure of the takeover rules for this repo

Reply:
I _think_ it's as easy as rebasing your PR on this branch and doing a PR here. If all you were doing is updating some text and fixing links, should be an easy merge and then we can continue/finish the conversation here... right?

Reply:
Yes, you should be able to do a PR against my PR. Then we approve yours first, then merge mine. But I'm back now, so if you want me to make the changes directly in my branch I can.